### PR TITLE
Added logic for Form Component

### PR DESCRIPTION
**Pull request by NehaSuthar**
Updated Files-- Form.js, App.js, Category.js

1. Updated App.js to pass SubmitHandler as a parameter for Form Component.
3. Added logic in Form Component to set the state with user-entered Quotes and category name and pass it to App.js via SubmitHandler.
4. Added logic to get the category image path from categoryList.js

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -1,10 +1,68 @@
...
+         quote2:'',
+         quote3:''
+      }
+   }
 
+   getCategoryImagePath(){  
+      const filteredcategory = categoryList.filter((category) => {
+            return(category.name === this.state.categoryName);
+         })
+      return filteredcategory[0];
```
Comment (epwallace) [inline src/components/Form.js:18]:
```suggestion
      const filteredCategory = categoryList.filter((category) => {
            return(category.name === this.state.categoryName);
         })
      return filteredCategory[0];
```

**Comment by epwallace** [comment]
Hi Neha, I checked out your branch and it looks great! One thing I'm wondering about: right now, the quotes are being stored in the state of `Form.js`. We will need to combine this data twice:

1. combine `quote1, quote2, quote3` with the quotes received from the quote API. I think this `Form.js` component is a good place for this logic. Maybe we should add this to our To Do list.

2. combine the quotes with the images from Unsplash. 

Right now, I am storing the `imageArray` from the Unsplash call in the `App.js` state. I was thinking we could pass `imageArray` and `quoteArray` as props into `Slideshow.js` and perform the logic to create the slideshow there.  

@darienweise @ChiaKuo @NehaSuthar let me know what you think!

**Comment by NehaSuthar** [comment]
I am storing the Quotes and Category in Form Component State and passing it to App Component via SubmitHandler function, By doing so Quotes and categoryName are available in App Component and we can pass these to the Slideshow component from there for our API call.

**Comment by epwallace** [comment]
Ohhhh, I see, I was expecting to see the quotes in the state when I used the React Developer Tools, but it's not implemented yet. Makes sense!

**Comment by epwallace** [comment]
I think it's ready for merging. 👍

**Comment by NehaSuthar** [comment]
ok...merging the code.

**Pull request merged**